### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by atmakurramakanth**

### CODE REVIEW

File: src/AllAssignmnets/Assignment1/Assignment1.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public static void main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        while(true){
+            System.out.println("Enter Regular Expression: ");
+            String regularExp = sc.nextLine();
+            Assignment1 assignment = new Assignment1();
+            assignment.fileSearch("/home/ramaa/IdeaProjects", regularExp);
+        }
+    }
+    public void fileSearch(String filePath,String regularExpression) throws  NullPointerException{
```
Comment (Keerthi-Gundla) [inline src/AllAssignmnets/Assignment1/Assignment1.java]: Please follow SOLID principles. This is voilating the single responsibility principle.

File: src/AllAssignmnets/Assignment2/Assignment2.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.*;
+
+public class Assignment2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.next();
+        System.out.println(checkString(input.toLowerCase()));
+    }
+
+    public static String checkString(String input){
```
Comment (Keerthi-Gundla) [inline src/AllAssignmnets/Assignment2/Assignment2.java]: This is voilating the single responsibility principle. Please follow SOLID principles.

**Review by Keerthi-Gundla — CHANGES_REQUESTED** [review]

File: src/AllAssignmnets/Assignment1/Assignment1.java (hunk 1 above)
Comment (atmakurramakanth) [reply src/AllAssignmnets/Assignment1/Assignment1.java]: solved

File: src/AllAssignmnets/Assignment2/Assignment2.java (hunk 2 above)
Comment (atmakurramakanth) [reply src/AllAssignmnets/Assignment2/Assignment2.java]: solved

**Review by Keerthi-Gundla — APPROVED** [review]

**Pull request merged**